### ISSUE TITLE
Version 3: Reviews, Issues, Suggestions By Mr.Pdtech

### ISSUE THREAD

**Issue by MrPdtech** [ankit/stylebot#554]
**Stylebot** is undoubtedly one of my best chrome extensions. However, I love the most recent update(version 3). but changing text and/or background colors to White has become a pain in the ass with this recent update. i literally have to be typing #FFFFFF all the freaking time. if it is possible please bring back the old color wheel (which most of us are already used to) but if this one is better then leave it as it is. we'll learn it. but most importantly, position the color wheel (perhaps at the center) so it'd be easier selecting colors that are at the edges of the color wheel. 

Again, i discovered that the green border that surrounds a selected object is missing. This feature is very helpful when editing cos you easily detect which property you're editing. i would love to see it back but this time smaller in size (1px) and perhaps a pixel off the edges of the object (for visibility purpose).

**SUGGESTION 1: OPACITY**
Please if it is possible, add the option for opacity(adjustable using a slider fro 0 to 100). For instance, some of a website's properties may be selected but NOT directly editable (cannot be hidden nor altered) but can be manually be hidden using code ( Opacity: 0;).

**SUGGESTION 2: TRANSPARENCY**
I don't know if this is possible. having an option with which the background of a property can be made transparent. For example a text with a color background then using an option to remove the background (background opacity to zero or perhaps hidden) leaving the text only without background.

**SUGGESTION 3: FONTS**
Please add the option to use custom fonts (fonts installed in computer).

**SUGGESTION 4: POSITIONING**
Is it possible to add an option to re-position  objects. for example moving a text some pixels (incrementally) to the right or left or both extreme ends. Margin and padding works "sometimes" but most times it doesn't just work.

That's all for now. Thank you!

**Comment by ankit** [comment]
Thanks for all the feature suggestions!

**Comment by MrPdtech** [comment]
> Thanks for all the feature suggestions!

You're most welcome sir. i hope to see these new features soonest and the old ones added back.

**Comment by SimonEast** [comment]
Yes +1 for opacity. 👍 I find myself dimming the opacity of elements that I don't want to get so much visual attention (sidebars and other promotional fluff).

To be even more amazing, it'd be great to have a "Restore opacity on hover" checkbox which basically applies this CSS automatically:

```
.element { opacity: 0.2; } /* or whatever your chosen opacity is */
.element:hover { opacity: 1; }
```

I find myself manually typing out this pattern quite regularly and would love to have it achieved more easily. It makes several websites much more pleasant to browse, while still being able to access content by hovering over it.
